pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1057,9 +1057,9 @@ BPF_HASH(module_init_map, u32, kmod_data_t, 256);                  // holds modu
 BPF_LRU_HASH(fd_arg_path_map, fd_arg_task_t, fd_arg_path_t, 1024); // store fds paths by task
 BPF_LRU_HASH(bpf_attach_map, u32, bpf_attach_t, 1024);             // holds bpf prog info
 BPF_LRU_HASH(bpf_attach_tmp_map, u32, bpf_attach_t, 1024);         // temporarily hold bpf_attach_t
-BPF_PERCPU_ARRAY(event_data_map, event_data_t, 1);                 // persist event related data
+BPF_PERCPU_ARRAY(event_data_map, event_data_t, 2);                 // persist event related data
```

Comment:
We will need to think of a solution for the `bufs` map as well, so path strings for example will not collide

Reply:
I'm adding a TODO (since the network code, the only one we know for sure is causing eBPF code preemption) does not use bufs map.

https://github.com/aquasecurity/tracee/issues/2618